Makefile

```diff
@@ -110,6 +110,10 @@ po:
 	msgen locale/en_GB/LC_MESSAGES/django.po -o locale/en_GB/LC_MESSAGES/django.po
 	msgen locale/en_GB/LC_MESSAGES/djangojs.po -o locale/en_GB/LC_MESSAGES/djangojs.po
 
+.PHONY: mo
+mo:
+	$(VIRTUAL_ENV)/bin/python manage.py compilemessages
+
 .PHONY: compilemessages
```

Comment:
can't we remove this then? I don't think we use it in any deployment script?

Reply:
Hm might be. Will check and remove accordingly.